projects/melon-core/src/partials/modules/fs/_moveFile.ts

```diff
@@ -0,0 +1,5 @@
+function _moveFile(from: string, to: string) {
+    _$internalBinding["MoveFile"](from, to, true);
```

Comment:
Isn't this the exact same bind as `MoveFileAsync`? Where do they differ?

Reply:
Yes, but the async ones will be turned into promise generators after babel compilation, due to [regenerator runtime](https://github.com/facebook/regenerator)